src/main/web/assets/js/app/config.json

```diff
@@ -0,0 +1,6 @@
+{
+	"api_url": "",
+	"debug": false,
+	"services": ["ASNMETA", "CUCKOO", "DNSMETA", "GOGADGET", "OBJDUMP", "PEID", "PEINFO", "VIRUSTOTAL", "YARA", "ZIPMETA"],
```

Comment:
You are fully correct here. The `config.json` file was carried over from the old front end so we can add the other front end features later easily. This data can later be received from storage, but we would have to add this as an API call.